tests/ijby/test-func-unary.py

```diff
@@ -136,31 +136,32 @@ def test_dt_pos_invalid(src):
 
 
 #-------------------------------------------------------------------------------
-# isna()
+# isna(): both dt.isna and dt.math.isna
```

Comment:
Don't know :) Then we also need to remove it from documentation, etc. I will review its usage.

Reply:
We can probably leave `dt.isna` available, just re-define is as the same function as `dt.math.isna`, this will be have backward-compatible while avoiding the need to maintain and test 2 functions doing the same (or subtly different?) thing.

Reply:
Ok, I redefined both `dt.isna` and `dt.abs` with their `dt.math` analogs.